### DISCUSSION TITLE
Options for writing on the Canvas

### DISCUSSION THREAD

**Discussion by lgarcia-sfisas** [razztyfication/vue-drawing-canvas#17] [Ideas]
Good afternoon friends,

Do you plan in the near future to have the option of being able to write on Canvas ?
You have a very complete tool and I give you my sincere congratulations, I am surprised that you do not have these options.

Thank you for your attention.

**Comment by razztyfication** [comment]
Hello, thanks for your kind words.
If what you mean is typing on canvas, I have no plans for that considering the beginning I made this package is just to draw.
But if you want to add text by typing on textfield or somewhere else and add it to canvas, you can use "**_additionalImages_**".
